LEGO1/lego/legoomni/src/actors/ambulance.cpp

```diff
@@ -222,9 +222,46 @@ AmbulanceMissionState::AmbulanceMissionState()
 	m_score5 = 0;
 }
 
-// STUB: LEGO1 0x10037440
+// FUNCTION: LEGO1 0x10037440
 MxResult AmbulanceMissionState::Serialize(LegoFile* p_legoFile)
 {
-	// TODO
-	return LegoState::Serialize(p_legoFile);
+	LegoState::Serialize(p_legoFile);
```

Comment:
can you rename `p_legoFile` to `p_file` in all instances?  We should never put the "lego" or "mx" prefix in the variable name. I know this was done like that in the existing state/serialization cases, but it should be changed there as well